play_scraper.py

```diff
@@ -1,29 +1,57 @@
-from bs4 import BeautifulSoup
 from urllib.request import urlopen, urlretrieve
+import os
+from bs4 import BeautifulSoup
+
 
 class PlayPageScraper:
-	def __init__(self, location):
-		self._base_url = "https://play.google.com/store/apps/details?id="
-		self._storage_dir = location
-		
-	def get_html(self, app_id, language=None):
-		url = self._base_url + app_id
-		
-		if language is not None:
-			url += "&hl=" + language
-		
-		page = urlopen(url)
-		soup = BeautifulSoup(page, 'html.parser')
-		
-		return soup
-		
-	def get_icon(self, app_id, language=None, directory=""):
-		html = self.get_html(app_id, language)
-		
-		icon = html.find(class_="T75of sHb2Xb")
-		src = icon["src"]
-		
-		location = self._storage_dir + directory
-
-		_, _ = urlretrieve(src, location + "/icon_" + app_id)
+    """A class for scraping Google Play Store web pages.
+
+    Attributes:
+        _base_url: str
+            Base url of the app's web pages.
+        _storage_dir: str
+            The main storage directory for retrieved info.
+    """
+
+    ICON_CLASS = "T75of sHb2Xb"
+
+    def __init__(self, base_url, storage_dir):
+        self._base_url = base_url
+
+        if not os.path.exists(storage_dir):
+            os.mkdir(storage_dir)
+        self._storage_dir = storage_dir
+
+    def __get_html(self, app_id, language=None):
```

Comment:
One underscore is almost always enough and using two [can be harmful](https://stackoverflow.com/a/1301369). It  also does not really make things more private and just mangles the name. This means someone who really wants to access it still easily can do so, it just makes the code less readable. As such it is also [discouraged in the style guide](https://google.github.io/styleguide/pyguide.html#3162-naming-conventions).

For these reasons when people say "make something private" in a Python-context they basically always refer to just one underscore.

```suggestion
    def _get_html(self, app_id, language=None):
```